feedly/api_client/data.py

```diff
@@ -72,9 +78,25 @@ class TagBase(Streamable):
     def tag_entry(self, entry_id:str):
         self._client.do_api_request(f'/v3/tags/{quote_plus(self["id"])}', method='put', data={'entryId': entry_id})
 
-    def tag_entries(self, entry_ids: List[str]):
-        self._client.do_api_request(f'/v3/tags/{quote_plus(self["id"])}', method='put',
-                                    data={'entryIds': [entry_id for entry_id in entry_ids]})
+    def tag_entries(self, entry_ids: List[str], errors_to_ignore: Set[int] = None):
+        """
+        Tag the given entries.
+        Will ignore given error_codes (default is only ignore 409, that happens when a duplicate is already tagged)
+        """
+        if errors_to_ignore is None:
+            errors_to_ignore = {409}
+        for i in range(0, len(entry_ids), 50):
+            self._tag_entries_batch(entry_ids[i: i + 50], errors_to_ignore)
+
+    def _tag_entries_batch(self, entry_ids: List[str], errors_to_ignore: Set[int]):
+        try:
+            self._client.do_api_request(f'/v3/tags/{quote_plus(self["id"])}', method='put', data={'entryIds': entry_ids})
+        except HTTPError as e:
+            resp: Response = e.response
+            if resp.status_code in errors_to_ignore:
+                logger.warning(f"Got 409 Client Error while tagging batch of entries: {resp.text}")
```

Comment:
`logger.warning(f"Ignored error from API: {e}")` ? the error may not be a 409 if a custom set is passed